### ISSUE TITLE
Loss Behavior Comparison: Training Text2CAD from Scratch vs. Using Pre-Trained Weights

### ISSUE THREAD

**Issue by DongshuoH** [SadilKhan/Text2CAD#32]
Hello, thank you for your excellent work.
I encountered some questions when training Text2CAD.
I trained Text2CAD for 5 epochs. In the first training, I did not load the pre-trained checkpoint_path. The training results are as follows.

![Image](https://github.com/user-attachments/assets/42d30100-c3b6-4fd4-9568-f20c747c029a)

Then I loaded your public pre-trained weights "checkpoint for v1.0" and then trained the model. The training results are as follows.

![Image](https://github.com/user-attachments/assets/f6b343ae-8616-4727-8ca3-9a97ea993f06)

The confusing point is that the Loss trend of the two trainings is the same, and the Loss of the initial first epoch is the same. I think this is confusing because the pre-trained model has been trained. Normally, the loss of loading the first epoch of pre-training should be much lower than the loss of training the first epoch from scratch.
(In addition, because it took too long to load the entire dataset, I made a split file myself and loaded 1% of the training data, but it should not affect such problems)

**Comment by DongshuoH** [comment]
I have solved it. Thank you!
